scripts/node.sh

```diff
@@ -183,8 +191,10 @@ do
    p) blspass="${OPTARG}";;
    d) download_only=true;;
    D) do_not_download=true;;
+   M) multi_key=true;;
```

Comment:
Disable multi key before it’s actually deployed?

Reply:
default is off. only enable it with "-M" option, still testing.